lucene/licenses/morfologik-ukrainian-search-NOTICE.txt

```diff
@@ -0,0 +1,6 @@
+morfologik-ukrainian-search is a POS tag dictionary in morfolgik format adjusted for searching.
+It's part of dict_uk project (https://github.com/brown-uk/dict_uk)
+
+Note: to better fit into full-text search model this dictionary has all word forms in lower case but keeps lemmas for proper nouns in upper case.
+
+Licensed under GPL/LGPL, CC BY-NC-SA 4.0, and Apache License 2.0.
```

Comment:
Hmm; not a lawyer, but this should better state "Licensed under Apache License 2.0"... Those multiple licenses don't agree with each other -- at least it should state *or*, not *and*.